src/lib/util.js

```diff
@@ -0,0 +1,14 @@
+'use strict'
+var EventManager = require('./eventManager')
+
+module.exports = {
+  extend: function (destination, source) {
+    for (var property in source) {
+      destination[property] = source[property]
+    }
+  },
+
+  produceEvent: function (destination) {
```

Comment:
Better to call this something like `makeEventCapable`?

On the other hand: Especially because the functions `register` and `trigger` have extremely common names, it would perhaps be better to not just extend the object, but rather have a property called `event` that is the EventManager.

Then we would have `this.event.trigger('selectTab',  [cls])` and `otherObject.event.register('changed', function() { >>> })`.